README.md

```diff
@@ -34,19 +34,23 @@ make # Run code generators, linters, sanitizers and test suits
 
 ## Authentication
 
-The client is initialized with an access token and the users organization ID
-when using Axiom Cloud or with the URL of the deployment and an access token
-when using Axiom Selfhost. The organization ID can be omitted in case an API
-token is used.
+The client is initialized with an access token. The access token can be an API
+token retrieved from the settings page of the Axiom deployment or a personal
+token retrieved from the users profile page.
```

Comment:
Kind of confusing as Profile is also in Settings

Reply:
API token: settings -> API Token
Personal Token: Settings -> profile -> add personal token